cmd/connect.go

```diff
@@ -54,6 +54,7 @@ peers & swapping data.`,
 	cmd.Flags().BoolVarP(&o.Setup, "setup", "", false, "run setup if necessary, reading options from environment variables")
 	cmd.Flags().BoolVarP(&o.ReadOnly, "read-only", "", false, "run qri in read-only mode, limits the api endpoints")
 	cmd.Flags().BoolVarP(&o.RemoteMode, "remote-mode", "", false, "run qri in remote mode")
+	cmd.Flags().BoolVarP(&o.RemoteAlwaysAccept, "remote-always-accept", "", false, "when running as  a remote, accept all datasets sent")
```

Comment:
Hrm, I'm wondering if we can adjust `remoteAlwaysAccept` to instead be `remoteMaxSizeAccept`, and only allow either 0 (`remoteAlwaysAccept == false`) or -1 (`remoteAlwaysAccept == true`). We could later use that to replace the `allowedDagSize` const, but let's do that as a follow up.

We're planning to follow up in the future with additional configuration details that will invalidate assumption suggested by "always accept" (because additional things will be able to reject a `dsync`). I'd love to avoid users complaining "but I set `remoteAlwaysAccept` to `true`, shouldn't it _always accept_?"

Reply:
I was considering this flag to be temporary, as we're going to have better configuration in the future, and this whole entire feature is in flux for now. But I also agree that `remoteMaxSizeAccept` is better and more accurately reflects what the code is doing now. I'll switch to that.